do-release-tags

```diff
@@ -28,6 +28,8 @@ def fatal(msg):
     sys.exit(1)
 
 parser = argparse.ArgumentParser(prog=sys.argv[0])
+parser.add_argument("--create-missing-refs", help="Create refs that don't already exist",
```

Comment:
How about just `--autocreate`?   Actually I'm not sure why we don't just do this by default.  Hmm...I guess that I wanted to avoid having typos create branches automatically.

But perhaps in practice that sort of thing is better handled by validation elsewhere.  So if you want to just change it to do this by default that's fine by me.